deepchecks/tabular/checks/model_evaluation/performance_bias.py

```diff
@@ -0,0 +1,549 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""The performance_disparity_report check module."""
+import itertools
+from typing import Callable, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core.checks import DatasetKind
+from deepchecks.core.errors import DeepchecksProcessError, DeepchecksValueError
+from deepchecks.tabular import Context, SingleDatasetCheck
+from deepchecks.utils.performance.partition import partition_column
+from deepchecks.utils.typing import Hashable
+
+
+class PerformanceBias(SingleDatasetCheck):
+    """
+    Check for performance disparities across subgroups, while optionally accounting for a control variable.
+
+    The check identifies "performance biases": large performance difference for a subgroup compared
+    to the baseline performance for the full population. It is intended to be used for fairness analyses.
+
+    Subgroups are defined by the categories of a "protected" feature. Numerical features are first binned
```

Comment:
We usually say "segment", why not keep saying it instead of "subgroup"?

Reply:
IMO it's OK to use a different terminology, because these are groups with some external meaning (actual groups in the world), while segments has a DS/Business feel to it.